pom.xml

```diff
@@ -1856,6 +1884,20 @@
                                         <type>jar</type>
                                         <path>forbidden-internal.txt</path>
                                     </signaturesArtifact>
+                                    <signaturesArtifact>
+                                        <groupId>org.hibernate.search</groupId>
+                                        <artifactId>hibernate-search-build-config</artifactId>
+                                        <version>${project.version}</version>
+                                        <type>jar</type>
+                                        <path>hibernate-core-incubating-internal.txt</path>
+                                    </signaturesArtifact>
+                                    <signaturesArtifact>
+                                        <groupId>org.hibernate.search</groupId>
+                                        <artifactId>hibernate-search-build-config</artifactId>
+                                        <version>${project.version}</version>
+                                        <type>jar</type>
+                                        <path>hibernate-core-internal-packages-internal.txt</path>
+                                    </signaturesArtifact>
```

Comment:
I'd say using incubating or internal Hibernate ORM stuff in tests is mostly okay.
It doesn't affect users, so we have more leeway to fix problems.
And in the case of "incubating" we might even *want* to test how Hibernate Search behaves when using incubating features of Hibernate ORM.